src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -43,7 +50,7 @@ class EndpointConnectionStats(
 
     // Maps the compacted NTP timestamp found in an SR SenderInfo to the clock time (in milliseconds)
     //  at which it was transmitted
```

Comment:
is this comment still accurate? since the value is now `Pair<Long, Long>`? also, i'm not sure what that value is, but might be nice to create a descriptive `typealias` for it.

Reply:
I'll update the comment.

The value is "ssrc and timestamp", there isn't really a better name than that.

Reply:
`SsrcAndTimestamp` to me is still an improvement over `Pair<Long, Long>`

Reply:
Done.